First pass at risk effects page review (#1103)
ihmeuw/vivarium_research: docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

What is a risk effect?
++++++++++++++++++++++

> A risk effect is a measure how some variable (the risk factor) causally impacts

Reviewer: ```suggestion
A risk effect is a measure of how some variable (the risk factor) causally impacts
```

some outcome variable. 

Generally, we will use risk effects models to represent 
associations between two variables that are assumed to be causal and 
:ref:`risk correlation models <2017_risk_models>` to represent *non-causal* 
associations between two variables (see the :ref:`causality <general_dags>`

> documentation page for more information on causality. It is important to note

Reviewer: ```suggestion
documentation page for more information on causality). It is important to note
```

and absolute terms. The risk ratio, the rate ratio, and the odds ratio are
relative measures of effect. Risk difference is an absolute measure of effect
and it is calculated by subtracting the risk of the outcome in exposed group
from unexposed [Measure_of_effect]_. See the 
:ref:`measures of excess risk page <excess_risk_measures>` for more 
information on such measures.

Risk effect in GBD
^^^^^^^^^^^^^^^^^^

The measure of risk effect in GBD is usually reported in relative term, namely
relative risk. It describes the relative relationship between the risk of
disease Y in the presence of agent X versus in absence of X. Mathematically,
it's calculated by dividing the rate of some outcome (such as incidence or mortality

> rate) population exposed to a risk by the outcome rate in unexposed population.

Author: This sentence was indeed a grammatical mess. Cleaned this up and will do a closer copy edit of the whole doc. Thanks

For example, if there are A incident cases and B person-years in exposed group;
C incident cases and D person-years in unexposed group, then the relative risk
(rate ratio) equals :math:`\frac{AD}{BC}`. The "unexposed" or "reference" group
in the GBD-estimated relative risks is always the TMREL for that risk. For GBD 
risk factors with continuous risk exposures, the GBD-estimated relative risks
represent the relative risk associated with a defined unit increase in risk exposure
above the TMREL. This unit increase may be more than a single unit change in risk
exposure, so it is important to clarify this with the GBD modeler.

.. todo::

    Determine if there is a data source that documents these units

> The GBD-estimated relative risks are adjusted for confounding and therefore

Reviewer: Not that we need to break down how to mathematically adjust for confounding -- we should just note above that that is how it is calculated *in the absence of confounders*.

> The GBD-estimated relative risks are adjusted for confounding and therefore

Author: Good point, I changed this sentence to reflect that and also added that GBD does an assessment of risk/outcome pairs to make sure they satisfy causal criteria (like absence of reverse causation). I didn't want to get too in the weeds here because I'm thinking it's covered in the burden of proof papers that are linked and the existing vivarium research causal inference page.

assumed to represent the *causal* effect size between the risk and outcome.
See the :ref:`causality documentation page <general_dags>` for more information 
on the topic. GBD has begun to quantify the quality of evidence for causality between
risk-outcome pairs they model in an effort termed "the burden of proof," summarized
in publications such as [Zheng-et-al-2022]_ and 
`this webpage <https://vizhub.healthdata.org/burden-of-proof/>`_. Additionally, the 
GBD-estimated relative risks represent the *total* effect of the risk on the outcome, 
including any mediated effects. See the :ref:`mediation documentation page <risk_mediation>` 
for more information on the topic.

While relative risks in GBD are typically age- and sex-specific, they are assumed 

> not to vary by location or year. GBD applies risk effects to either YLDs, YLLs, or both.

Author: This is a good question. I haven't seen one, but I haven't systematically checked. I'll poke around and see if I can find anything.

> not to vary by location or year. GBD applies risk effects to either YLDs, YLLs, or both.

Author: Ok, I think that there are no cases of this, although there appear to be some inconsistencies with the occupational noise and childhood sexual abuse risk factors in the RR database and GBD compare that made it confusing to figure out. I feel safe concluding that this does not happen in GBD currently, though.

 - :math:`p(e)` is probability density function used to calculate the probability 
   of given risk exposure level e

We can refer to the outcome rate multiplied by (1 - PAF) as the "risk-deleted outcome rate."

.. todo::

    Add a note about bias this introduces...

        PAF relies on exposure in the population, not the "at-risk" group for the outcome. This bias is larger when the at-risk population is small relative to the total population.

        But maybe this belongs in the PAF section?

Direct and indirect risk effect
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

.. todo::

    Move mediation-related information to the mediation page

For a risk-mediator outcome, simulation model would map a probability
distribution of possible mediator exposure level to each measurement of
associated risk factor (e.g. there is X% chance you will observe a SBP
>= 100 mm Hg for given BMI of 25 in adults).

> In general, we would model the risk-outcomes that is directly correlated

Reviewer: ```suggestion
In general, we would model the risk-outcomes that are directly causally related
```

> (e.g. BMI -> IHD), but sometimes we consider add mediator to account for

Reviewer: ```suggestion
(e.g. BMI -> IHD), but sometimes we consider adding a mediator to account for
```
